libs/shared/contentful/src/__generated__/graphql.d.ts

```diff
@@ -0,0 +1,2870 @@
+import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core';
```

Comment:
Did you double check with SubPlat that this diff should have been committed elsewhere? 🤔

Reply:
This should not be here. Removing.